fauxfactory/__init__.py

```diff
@@ -212,10 +211,10 @@ def gen_cjk(length=10):
     # valid range of CJK codepoints is 0x4E00 - 0x9FCC, inclusive. Python 2
     # and 3 support the `unichr` and `chr` functions, respectively.
     codepoints = [random.randint(0x4E00, 0x9FCC) for _ in range(length)]
-    try:
-        # (undefined-variable) pylint:disable=E0602
+    if sys.version_info.major == 2:
+        # pylint:disable=undefined-variable
```

Comment:
I have recently discovered that it is possible to name warnings directly, rather than by their codes. Cool.